engine/sound/src/devices/device_openal.cpp

```diff
@@ -139,20 +151,27 @@ namespace dmDeviceOpenAL
 
             if (iter > 1000) {
                 dmLogError("Still buffers in OpenAL. Bailing.");
+                break;
```

Comment:
If we've waited so long, we might as well at least try to continue the shutdown sequence.